src/site/content/en/blog/largest-contentful-paint/index.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Largest Contentful Paint
+subhead: An important, new user-centric performance metric
+authors:
+  - philipwalton
+date: 2019-08-08
+hero: hero.jpg
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+hero_position: top
+alt: Artist painting tools
+description: |
+  This post introduces the Largest Contentful Paint API, how to use it,
+  and offers advice on how to optimize your pages for this metric
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+  - metrics
+---
+
+Historically, it's been a challenge for web developers to measure how quickly
+the main content of a web page loads and is visible to users.
+
+Older metrics like
+[load](https://developer.mozilla.org/en-US/docs/Web/Events/load) or
+[DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/Events/DOMContentLoaded)
+are not good because they don't necessarily correspond to what the user sees on
+their screen. And newer, user-centric performance metrics like [First Paint
+(FP) and First Contentful Paint
+(FCP)](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics#first_paint_and_first_contentful_paint)
```

Comment:
You don't need to include abbreviations if you're not actually using them. In fact, I'd take them out. Check this throughout.

Reply:
In this case I think it's good to include them. This page will ultimately be part of a group of metrics content, and these metrics are often referred to by just their acronyms, so I think it's helpful to keep them (especially if someone's seen just the acronyms elsewhere and didn't know what it meant).